src/_includes/partials/links.njk

```diff
@@ -0,0 +1 @@
+<link rel="stylesheet" href="/assets/base.css">
```

Comment:
Please remove the leading slash from the CSS file path. This should potentially fix the theming issue.

Reply:
Removed the leading slash 20beecb89faa45b0afcbf7855930fbca4160cf98